fiftyone/server/view.py

```diff
@@ -225,6 +234,13 @@ def get_extended_view(
     if sort_by:
         view = view.sort_by(sort_by, reverse=bool(desc), create_index=False)
 
+    for stage in view._stages:
+        if isinstance(stage, fosg.GroupBy):
+
+            view = view.mongo(
+                [{"$addFields": {"_group": stage._get_group_expr(view)[0]}}]
```

Comment:
any concern that `_group` might be used as a real field by users?

Reply:
we reserve underscore fields for internal use